membershipextras.php

```diff
@@ -157,13 +157,19 @@ function membershipextras_civicrm_pre($op, $objectName, $id, &$params) {
    * options for now.
    */
   static $contributionID = NULL;
+  static $preEditContributionHook = NULL;
   if ($op === 'edit' && $objectName === 'Contribution') {
     $contributionID = $id;
+
+    $preEditContributionHook = new CRM_MembershipExtras_Hook_Pre_Contribution($id, $params);
+    $preEditContributionHook->updatePaymentPlanStatus();
```

Comment:
Why did  you choose to update the recurring contribution's status before the contribution is saved? Wouldn't it make more sense to do it after, as in the contribution's post hook?